jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/SimpleRuntimeEnvironment.java

```diff
@@ -106,8 +109,30 @@ public void init() {
      * @param type type of the asset
      */
     public void addAsset(Resource resource, ResourceType type) {
-        this.kbuilder.add(resource, type);
-        if (this.kbuilder.hasErrors()) {            
+        if (resource.getSourcePath() != null ) { 
+            String path = resource.getSourcePath();
+           
+            String typeStr = null;
+            if( path.endsWith(".csv") ) { 
```

Comment:
It might be better to check the file extension against org.kie.api.io.ResourceType.DTABLE before building a DecisionTableConfigurationImpl. Furthermore you could use the DTABLE.defaultExtension to differentiate between XLS and CSV... Just my 2c... saves hard coding file extensions when KIE has a Registry of them already....